app/assets/javascripts/vanilla_nested.js

```diff
@@ -85,6 +86,7 @@
   // "wrapper" is the wrapper to hide
   // "element" is the link to remove the wrapper
   function hideFieldsWithUndo(wrapper, element) {
+    wrapper.classList.add('hide-by-vanilla-nested');
```

Comment:
```suggestion
    wrapper.classList.add('hidden-by-vanilla-nested');
```

To be in past tense like the rest